docs/units/index.rst

```diff
@@ -82,6 +82,15 @@ Units that "cancel out" become a special unit called the
 
     >>> u.m / u.m
     Unit(dimensionless)
+    
+To create a simple :ref:`dimensionless quantity <doc_dimensionless_unit>`, multiply 
+or divide a value by the unscaled dimensionless unit::
+
+    >>> q1 = 1.0 * u.dimensionless_unscaled
+    >>> # or equivalently, though less preferred for readability:
```

Comment:
leave comments out of code example, in general examples should show one new behaviour, stick with one of them or add two separate examples.

Reply:
Good to know, thanks